test/freenet/clients/http/RssSniffTest.java

```diff
@@ -0,0 +1,39 @@
+package freenet.clients.http;
+
+import junit.framework.TestCase;
+
+/**
+ * Test cases for the RSS feed sniffing logic in FProxyToadlet.
+ */
+public class RssSniffTest extends TestCase {
```

Comment:
And I think we already have a dependency on JUnit 4, so just drop the `extends TestCase` and annotate the test methods with `@Test`, please? :)